tests/data_protection/oadp/test_velero.py

```
@@ -84,13 +90,13 @@ def test_restore_multiple_namespaces(
 def test_backup_vm_data_volume_template_with_datamover(
     rhel_vm_with_data_volume_template, velero_restore_first_namespace_with_datamover
 ):
-    check_file_in_vm(vm=rhel_vm_with_data_volume_template)
+    check_file_in_vm(vm=rhel_vm_with_data_volume_template, file_name=FILE_NAME_FOR_BACKUP, file_content=TEXT_TO_TEST)
 
 
 @pytest.mark.s390x
 @pytest.mark.polarion("CNV-10589")
 def test_restore_vm_with_existing_dv(rhel_vm_from_existing_dv, velero_restore_second_namespace_with_datamover):
```

Ruff (0.14.10)

Unused function argument: `velero_restore_second_namespace_with_datamover`

(ARG001)